one-ui/src/main/java/com/marklogic/hub/curation/controllers/CollectionsController.java

```diff
@@ -0,0 +1,4 @@
+package com.marklogic.hub.curation.controllers;
+
+public class CollectionsController {
```

Comment:
This seems like it can be deleted?

Reply:
my bad, something wrong during commit and push.

Reply:
This is still in the PR, it should be removed, right?

Reply:
Let's put this in a separate PR for a separate sub-task under 3991. It needs a test for it, and we should instead use a new DS endpoint instead of continuing use of REST endpoints (and the mlCollections REST extension is doing too many things too, it's not just used for retrieving collections).